src/thunderbots/software/backend_output/main.cpp

```diff
@@ -50,6 +50,8 @@ int main(int argc, char** argv)
     // We loop at 30Hz so we don't overload the network with too many packets
```

Comment:
While we're here, can we make this change?
```suggestion
    // We loop at a set rate so we don't overload the network with too many packets
```